proxy/pkg/config/config.go

```diff
@@ -148,7 +181,7 @@ func (c *Config) ParseEnvVars() (*Config, error) {
 
 	log.Infof("Parsed configuration: %v", c)
 
-	return c, nil
+	return c, err
```

Comment:
I'd prefer to keep `nil` here since it is more clear that there will always never be an err at this point